recite/models.py

```diff
@@ -69,28 +68,9 @@ def __str__(self):
 def auto_delete_files_on_delete(sender, instance, **kwargs):
     """
     Deletes Recitation files from filesystem
-    when corresponding `Reciter` object is  deleted.
+    when the corresponding `Reciter` object is deleted.
     """
     reciter_path = os.path.join(
         settings.MEDIA_ROOT, 'recite', str(instance.slug))
     if os.path.exists(reciter_path):
         shutil.rmtree(reciter_path)
-
-
-@receiver(models.signals.pre_save, sender=Reciter)
```

Comment:
Why did you decide to delete this method?

Reply:
If we try to change Reciter in admin panel, the Recitations are recreated but the older ones are not deleted. It was like quick fix, really don't think these operations will be used often.